application/controllers/QuestionEditorController.php

```diff
@@ -590,7 +590,7 @@ public function actionGetQuestionPermissions($iQuestionId = null)
             "editorpreset" => App()->session['htmleditormode'],
             "script" =>
                 Permission::model()->hasSurveyPermission($oQuestion->sid, 'surveycontent', 'update')
-                && SettingsUser::getUserSetting('showScriptEdit', App()->user->id),
+                && SettingsUser::getUserSetting('showScriptEdit', App()->user->id) == 1,
```

Comment:
SettingsUser::getUserSetting is an object

`(SettingsUser::getUserSetting('showScriptEdit', App()->user->id) && SettingsUser::getUserSetting('showScriptEdit', App()->user->id)->stg_value)`

Reply:
sorry,
should be
&& SettingsUser::getUserSettingValue('showScriptEdit', App()->user->id) == 1,

because  after set my account->Show script field option 
SettingsUser::getUserSetting('showScriptEdit', App()->user->id) 100% is return not null, showScriptEdit is 0 or 1

Reply:
Please : set debug > 2 … 

And look at https://github.com/LimeSurvey/LimeSurvey/blob/108ddbf3db579b012082b8f937f8035b13f1c51f/application/models/SettingsUser.php#L137

it return NULL or Object

Reply:
i know,I resubmitted